client/account-recovery/lost-password/test/index.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import reactTapEventPlugin from 'react-tap-event-plugin';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import { LostPassword } from '..';
+
+describe( 'LostPassword', () => {
+	it( 'should render as expected', () => {
+		const wrapper = shallow( <LostPassword className="test__test" translate={ noop } /> );
```

Comment:
[Lodash's `_.identity`](https://lodash.com/docs/4.16.4#identity) might be a more useful stand-in for `translate` than `noop`.